### PR TITLE
VMware new additions for deploy-as-is

### PR DESCRIPTION
**Pull request by nvazquez**
VMware documentation for extension to virtual appliances and deploy-as-is

### CODE REVIEW

**Comment by nvazquez** [comment]
require docbuild

**Comment by nvazquez** [comment]
requesting docbuild

File: source/adminguide/templates.rst [hunk 1]
```diff
@@ -237,7 +249,7 @@ To upload a template:
       available, or All Zones to make it available throughout
       CloudStack.
 
-   -  **OS Type**: This helps CloudStack and the hypervisor perform
+   -  **OS Type**: (Except fo VMware). This helps CloudStack and the hypervisor perform
```
Comment (shwstppr) [inline source/adminguide/templates.rst:252]:
```suggestion
   -  **OS Type**: (Except for VMware). This helps CloudStack and the hypervisor perform
```

File: source/adminguide/virtual_machines/virtual_appliances.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+VMware templates can provide different deployment options in their OVF descriptor file. CloudStack obtains
+the different deployment options when the template is registered and it displays them to the users
+in the virtual machine deployment wizard, under the 'Compute Offering' section.
+
+After the user selects a deployment option, CloudStack lists the compute offerings which match or exceed the
+deployment options hardware requirements for CPU and memory.
+
+.. note::
+    All the custom unconstrained compute offerings are displayed, but those constrained custom offerings
```
Comment (shwstppr) [inline source/adminguide/virtual_machines/virtual_appliances.rst:20]:
```suggestion
    All the custom unconstrained compute offerings are displayed, but only those constrained custom offerings
```

**Review by shwstppr — COMMENTED** [review]
Minor nits

**Comment by rhtyd** [comment]
@nvazquez can you rebase to 4.15 branch of the docs repo?

**Comment by nvazquez** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
I don't disk/nic controllers being mentioned as inherited from the OVF and not being possible to specify it any more. The same for bios vs uefi and anything else (if anything else..?)

**Comment by nvazquez** [comment]
Done @andrijapanicsb

**Comment by nvazquez** [comment]
requesting docbuild

**Pull request merged**